### PR TITLE
Update loiter-to-qland-mode.rst

### PR DESCRIPTION
**Pull request by magicrub**
This updates the docs to match new behavior from PR https://github.com/ArduPilot/ardupilot/pull/25318

### CODE REVIEW

**Comment by magicrub** [comment]
Does this need a "only applies to version 4.5+"?

**Comment by Hwurzburg** [comment]
no....once its code is added, I will link this PR in wiki futures...it wont merge until 4.5 beta is released

File: plane/source/docs/loiter-to-qland-mode.rst [hunk 1]
```diff
@@ -4,9 +4,9 @@
 LOITER to Altitude and QLAND Mode
 =================================
 
-If the previous mode was a fixed wing mode, this mode will perform a descending fixed wing LOITER down to :ref:`Q_RTL_ALT<Q_RTL_ALT>` and then switch to :ref:`QLAND mode <qland-mode>`. The LOITER center is set to the point when entering the mode. 
+If the previous mode was a fixed wing mode, this mode will perform a descending fixed wing LOITER down to :ref:`Q_RTL_ALT<Q_RTL_ALT>` and then switch to :ref:`QLAND mode <qland-mode>`. If the aircraft was already performing a loiter when switching to this mode the LOITER center does not change, otherwise the LOITER center will become the same location it is at when entering the mode.
 
-.. note:: the QLAND target will be directly below the point it switches to QLAND (ie on the loiter circle path, not its center)
+.. note:: the QLAND target will be directly below the point it switches to QLAND (ie on the loiter circle path, not its center) if it is not already loitering
```
Comment (Hwurzburg) [inline plane/source/docs/loiter-to-qland-mode.rst:9]: this is not correct as I understand the pr... Your PR just keeps the loiter circle from moving if already in loiter...does not affect where the qland point is

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: plane/source/docs/loiter-to-qland-mode.rst (hunk 1 above)
Comment (magicrub) [reply plane/source/docs/loiter-to-qland-mode.rst:9]: Oh, right. The QLand point behavior is unchanged. It's the second PR/hack that does that. I'll fix it up

Comment (magicrub) [reply plane/source/docs/loiter-to-qland-mode.rst]: fixed!

**Pull request merged**